src/integration/blockchain/lightning/lightning-client.ts

```diff
@@ -34,11 +34,31 @@ export class LightningClient {
 
   constructor(private readonly http: HttpService) {}
 
+  // --- DFX API --- //
+  async getSignMessage(address: string): Promise<string> {
+    return this.http
+      .get<{ message: string }>(`${Config.dfxApiUrl}/auth/signMessage`, {
+        params: { address: address },
+      })
+      .then((m) => m.message)
+      .catch((e) => this.throwHttpException(e));
+  }

```

Comment:
I think I would move this to the LightningService, it doesn't interface with the Lightning hardware.